clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -458,26 +478,32 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
 
   private Closure updateRegionBackendServices(Compute compute, String project, String region, String backendServiceName, BackendService backendService) {
     return {
-      BackendService serviceToUpdate = compute.regionBackendServices().get(project, region, backendServiceName).execute()
+      BackendService serviceToUpdate = GoogleExecutor.timeExecute(
+          compute.regionBackendServices().get(project, region, backendServiceName),
+          "compute.regionBackendServices.get", TAG_SCOPE, SCOPE_REGIONAL, TAG_REGION, region)
```

Comment:
From browsing around here, it kind of looks like we could use some reflection to figure out an appropriate class name instead of passing the string each time. Haven't dug deeply yet though.